src/controllers/tasks/people.with.significant.control.controller.ts

```diff
@@ -15,23 +22,66 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
-export const post = (req: Request, res: Response, next: NextFunction) => {
+export const post = async (req: Request, res: Response, next: NextFunction) => {
   try {
     const pscButtonValue = req.body.pscRadioValue;
+    const companyNumber = getCompanyNumber(req);
+    const transactionId = req.params[urlParams.PARAM_TRANSACTION_ID];
+    const submissionId = req.params[urlParams.PARAM_SUBMISSION_ID];
+
+    if (!pscButtonValue) {
+      return res.render(Templates.PEOPLE_WITH_SIGNIFICANT_CONTROL, {
+        templateName: Templates.PEOPLE_WITH_SIGNIFICANT_CONTROL,
+        peopleWithSignificantControlErrorMsg: PEOPLE_WITH_SIGNIFICANT_CONTROL_ERROR,
+        backLinkUrl: urlUtils
+          .getUrlWithCompanyNumberTransactionIdAndSubmissionId(TASK_LIST_PATH, companyNumber, transactionId, submissionId),
```

Comment:
I think using `urlUtils.getUrlToPath(TASK_LIST_PATH, req)` instead would save you a few lines.